flytestdlib/contextutils/context.go

```diff
@@ -12,14 +12,15 @@ import (
 type Key string
 
 const (
-	AppNameKey         Key = "app_name"
-	NamespaceKey       Key = "ns"
-	TaskTypeKey        Key = "tasktype"
-	ProjectKey         Key = "project"
-	DomainKey          Key = "domain"
-	WorkflowIDKey      Key = "wf"
-	NodeIDKey          Key = "node"
-	TaskIDKey          Key = "task"
+	AppNameKey    Key = "app_name"
+	NamespaceKey  Key = "ns"
+	TaskTypeKey   Key = "tasktype"
+	ProjectKey    Key = "project"
+	DomainKey     Key = "domain"
+	WorkflowIDKey Key = "wf"
+	NodeIDKey     Key = "node"
+	TaskIDKey     Key = "task"
+	// Adding this label to a metric will cause higher cardinality. Use with caution.
```

Comment:
Nit: "this" is unclear to me:

```suggestion
	// Adding the ExecIDKey label to a metric will cause higher cardinality. Use with caution.
```